src/pudl/metadata/codes.py

```diff
@@ -1810,104 +1810,104 @@
     "core_eia__codes_balancing_authorities": {
```

Comment:
I ended up adding the mapping of region->interconnect to `core_eia__codes_balancing_authorities` . It's not the most strictly well normalized place to add this info, but it seemed fairly logical/straightforward

Reply:
Yeah these coding tables aren't perfectly normalized, but they're small and useful and I think it's okay.